### PR TITLE
Fix some warnings when using an older version of protobuf

### PR DESCRIPTION
**Pull request by lmoneta**
Fix protobuf warnings as suggested in https://github.com/protocolbuffers/protobuf/issues/7140#issuecomment-599467033

The warnings should however be fixed in the latest protobuf version, 3.17

### CODE REVIEW

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2004/default with flags -Dtmva-sofie=On

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2004/default with flags -Dtmva-sofie=On

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu2004/default with flags -Dtmva-sofie=On

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu18.04/default with flags -Dtmva-sofie=On

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu18.04/default, ROOT-ubuntu2004/default, ROOT-ubuntu16/default with flags -Dtmva-sofie=On

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu18.04/default, ROOT-ubuntu2004/default with flags -Dtmva-sofie=On

**Comment by lmoneta** [comment]
@phsft-bot build just on ROOT-ubuntu18.04/default, ROOT-ubuntu2004/default with flags -Dtmva-sofie=On

File: tmva/sofie/test/CMakeLists.txt [hunk 1]
```diff
@@ -37,8 +37,8 @@ target_include_directories(emitFromONNX PRIVATE
 )
 
 target_link_libraries(emitFromONNX ${Protobuf_LIBRARIES} ROOTTMVASofie)
-set_target_properties(emitFromONNX PROPERTIES
-  POSITION_INDEPENDENT_CODE TRUE)
+set_target_properties(emitFromONNX PROPERTIES POSITION_INDEPENDENT_CODE TRUE)
+target_compile_options(emitFromONNX PRIVATE -Wno-unused-parameter -Wno-array-bounds)
```
Comment (Axel-Naumann) [inline tmva/sofie/test/CMakeLists.txt:41]: Can you add `# not needed with protobuf version ...` so we know we can kick this out in a couple of years? Same to the other two occurrences, please.

**Review by Axel-Naumann — APPROVED** [review]
But see comment.

**Pull request merged**